### DISCUSSION TITLE
How to update app manifest?

### DISCUSSION THREAD

**Discussion by akhial** [saleor/saleor#13301] [Q&A] [answered]
Hi :wave: 

I installed a self-hosted app thru the Saleor dashboard, however after updating the app the version shown in the dashboard is not updated.
However I think this is only visual, as when I access the app it is using the new version.

What is the correct way to update an app? Do I have to re-install the app?

Thanks.

**Comment by lkostrowski** [comment, marked as answer]
Hey @akhial 

Right now, we don't have a full API that will allow updating fields like `version`. You can check the `appUpdate` mutation - it only accepts `name` and `permissions`.

In the future, we plan to support more fields, including version, so app developers will be able to perform full updating mechanisms.

If you want to refresh manifest data you need to reinstall the app.

**Reply by akhial** [reply]
Thanks Lukasz 👍
